tests/unit/CheckoutFormTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    define('_PS_VERSION_', 'TEST_VERSION');
+}
+
+class CheckoutFormTest extends PHPUnit_Framework_TestCase
+{
+    private $list_of_expiration_year;
+
+    public function setup()
+    {
+        $checkout_form = new CheckoutForm();
+        $this->list_of_expiration_year = $checkout_form->getListOfExpirationYear();
+    }
+
+    public function testGetListOfExpirationYear_getListOfExpirationYear_totalNumberOfExpirationYearMustBe11()
+    {
+        $this->assertEquals(11, count($this->list_of_expiration_year));
+    }
+
+    public function testGetListOfExpirationYear_getListOfExpirationYear_theBeginningOfExpirationYearMustBeCurrentYear()
+    {
+        $current_year = date('Y');
+
+        $beginning_of_expiration_year = $this->list_of_expiration_year[0];
```

Comment:
@nimid I think if we rename this test to 

`testGetListOfExpirationYear_retrieveTheFirstElementOfAnArray_theResultMustBeTheCurrentYear`

we can remove these `$current_year`, `$beginning_of_expiration_year` variables and just have only
`$this->assertEquals(date('Y'), $this->list_of_expiration_year[0]);`

It's still understandable.